server.js

```diff
@@ -23,6 +49,47 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20)
+  if (thoughts.length > 0){ 
+    res.json(thoughts)
+  } else {
+    res.json({ message: "No thoughts found" })
+  }
+
+})
+
+app.get('/thoughts/:tags', async (req, res) => {
```

Comment:
A better way to achieve it is to use queries, or change the endpoint to `/thoughts/tag/:tag`. Because intuition says that after `/thoughts` should be `:id`